### PR TITLE
Rs 2022 06 10

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 3/03_01_CIEL03_1.txt [hunk 1]
```diff
@@ -96,12 +105,14 @@ I'm glad to see everyone's in good shape... Wait,
 [sha:ddb30d6abde7e7a0169bfb600399af46fa0e0e9b]{
 -- Page 9, Offset 4871. Glued. Mods: @x, @n.
 -- あらら、欠席者が二人もいるのね。季節の変わり目だし、
- oh no, looks like two students are out. Well, it is the change of season after all.
+// Is this as much of a concept in EN?
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_1.txt:108]: not really but it's what it says so 🤷

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 2]
```diff
@@ -6,12 +6,13 @@
...
 -- Page 13, Offset 4881.
 -- しかし、今日はなんとなく食欲が湧かない。
-However, my appetite just isn't coming today.
+However, I just don't have an appetite today.
 }
 [sha:ace5b88dc2685fae28ef36496784183b8083c6ca]{
 -- Page 13, Offset 4882.
 -- こういう時に食べると決まって<嘔|も><吐|ど>してしまう事になるし、教室で体を休めている方が得策な気がするが……。
-There's no way I won't end up vomiting if I eat like this. I think I'll be better off resting in the classroom...
+If I force myself to eat when I'm feeling this way I'll just end up being sick later,
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:14]: way, I'll just

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 3]
```diff
@@ -26,17 +27,21 @@ There's no way I won't end up vomiting if I eat like this. I think I'll be bette
...
+// Not _strictly_ what 濃い顔 seems to mean
+// https://www.weblio.jp/content/%E9%A1%94%E3%81%8C%E6%BF%83%E3%81%84
+Any appetite I may have had disappears the second I see his ugly mug.
 }
 [sha:416ec3d122a0af71b689fe66f6cdfc00c4c6ddae]{
 -- Page 14, Offset 4886.
 -- よし、昼食は抜こう。きっかけをクソありがとう有彦。
-Okay, I'll skip lunch today. Thanks for the excuse, Arihiko.
+// One piece reference?
+// http://xn--zcktc0dqa0c7959ah6we.com/archives/77
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:38]: wow, an actual punycode url in the wild! Don't see that every day

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 4]
```diff
@@ -131,19 +141,19 @@ Alright, I'll give you some of my sweet bread then!"
...
 -- んじゃ遠慮なくいただきます」
-// it seems like いただきます isn't particularly consistently translated.
-// I'll leave this one for now.
  Well, I'll be digging in then."
 }
 [sha:e5fa6eae7bd3e1b0c2734eb305a8a9e09c225167]{
 -- Page 21, Offset 4907.
 -- 有彦は両手を合わせてから菓子パンを食べ始める。
-Arihiko clasps his hands together and begins to eat his bread.
+// Not exactly what he's doing here, but avoids the culture gap a bit
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:149]: I really don't think this localization is necessary. anime folk know what clapping your hands together means and I don't really like removing culture stuff like this if possible.

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 5]
```diff
@@ -131,19 +141,19 @@ Alright, I'll give you some of my sweet bread then!"
...
+Arihiko rubs his hands together then starts to tuck into his bread.
 }
 [sha:fa453996d3a66379440a556419215cbd1b414d34]{
 -- Page 21, Offset 4908.
 -- こう見えて、感謝のいただきますとごちそうさまを言える男なのだった。
-Seeing him like this, he really seems like a man who %{i}appreciates%{/i} his meals.
+// Also a bit hard to translate without baggage
+Despite his appearance, he's still a man who takes his time to appreciate his meals.
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:156]: revert "a man who takes his time to appreciate his meals."

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 6]
```diff
@@ -199,22 +209,25 @@ Wait a minute, what are you making me say, Tohno-kun!?"
...
 -- Page 27, Offset 4921.
 -- ボランティアの時はもう、かなり無茶をしちゃってます」
-When I'm volunteering, I tend to barely scrape by."
+I have to push myself pretty hard to make it on time when I'm volunteering."
 }
 [sha:ee1bc3ce3055e81bc77b296e05e87e3b03949fd8]{
 -- Page 28, Offset 4922.
 -- 「わたしの実家はパン屋さんでしたから、すっごく朝が早かったんです。けどわたしはどうしても朝に弱くて。もう子供のころからお父さんに怒られてばかりでした」
-"My parents owned a bakery, so I used to wake up very early every day. But, no matter how I tried, I always struggled in the morning. My father has been driven mad by it ever since I was a kid."
+"My parents ran a bakery, so we had always had to get up incredibly early every day.
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:228]: "we had always had" -> "we always had"

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 7]
```diff
@@ -299,27 +314,33 @@ Arihiko nods in agreement.
...
 -- Page 34, Offset 4939.
 -- 「それで、朝が弱い先輩はたいてい学食だったわけなのか」
-"So, does a late riser like you normally go to the cafeteria?"
+"So, I suppose a late riser such as yourself ends up eating in the cafeteria most days?"
 }
 [sha:75998a4ebb0d0d3aaa11f9f2be14d80be0e4b2b1]{
 -- Page 35, Offset 4940.
 -- 「ええ、そういうわけなので学食ばかりですけど、たまにはこうやってお弁当ぐらい作ります。出来合ものの寄せ集めですけど。
-"Yeah, I eat in the cafeteria most of the time. I only make my own lunch like this every so often. Usually, it's just a bunch of pre-prepared dishes.
+"Yeah, I end up eating in the cafeteria more than I'd like to admit,
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:322]: admit.

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 8]
```diff
@@ -299,27 +314,33 @@ Arihiko nods in agreement.
...
+ but I still manage to make my own lunch like this every now and again.
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:323]: I still manage to make my own lunch like this now and again though.

File: script/Ciel/Day 3/03_01_CIEL03_2_3.txt [hunk 9]
```diff
@@ -368,12 +389,16 @@ The news has been saying the same thing since the third case, but in the end, th
...
-The fact these murders continue to occur to this extent in modern Japan is beyond abnormal and may even be considered paranormal.
+For a string of serial murders to continue this long in modern Japan is beyond abnormal.
+ It's practically bordering on paranormal.
 }
 [sha:2514f5dddb4e72f817a3a78a9d93e153d99ae18c]{
 -- Page 87, Offset 5055.
 -- 「……そうでした。現代の吸血鬼、なんてふざけたテロップがうたれてるから、身近な出来事の気がしなかったというか……たしかに軽率でした。俺だってこの街に住んでいるんだから、気をつけなくちゃいけないのに」
-"...You're right. The jokey banner on the news proclaiming it was the work of some 'modern vampire' made it seem like something I shouldn't concern myself with. But I live in this city too, so I should be taking it seriously."
+"...You're right.
+ I didn't really take it seriously after I saw the chyron calling it the work of a 'modern day vampire'...
```
Comment (Avuxo) [inline script/Ciel/Day 3/03_01_CIEL03_2_3.txt:399]: nobody knows what a chyron is, my browser thinks its a typo unless I make it a proper noun for the company name. revert it to "news banner" :aokoheh:

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 3/03_01_CIEL03_1.txt [hunk 10]
```diff
@@ -1,47 +1,50 @@
...
 -- Page 2, Offset 4855.
 -- そんな状況で、
-In that situation,
+With this as a backdrop―――
 }
 [sha:76840eea3936d066cb72bb9327f299407ac6ea7b]{
 -- Page 2, Offset 4856.
 -- 「……はあ。ずいぶんと余裕のあるご到着ですねぇ」
-"...%{i}Sigh%{/i}, you sure arrived with a lot of time to spare, huh?"
+"...%{i}Sigh%{/i}. You sure arrived with a plenty of time to spare, didn't you?"
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_1.txt:25]: remove a

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 11]
```diff
@@ -96,12 +103,14 @@ They complain that I'm sleeping in even if I wake up at seven o'clock in the mor
 [sha:f8d6d319b461f939871ed6b49a9a9b655b04c469]{
 -- Page 19, Offset 4899.
 -- 管理体制<万万歳|ばんばんざい>、若者はもっとタイトに生きねえとな。若いうちから楽しちゃろくな大人になれないぞ」
-Management like that is worthy of the highest praise. Young people need much stricter lives. If you have fun while you're young, you'll never become a decent adult."
+I've not nothing but praise for that sort of micromanagement.
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:106]: got

File: script/Ciel/Day 3/03_01_CIEL03_2_1.txt [hunk 12]
```diff
@@ -455,12 +481,13 @@ You'll ruin your body if you live like that."
 [sha:82070f01e2a5ba371aa462ea96c1dc00f57d6e71]{
 -- Page 48, Offset 4971.
 -- 医者に体重を増やさないように注意もされてるし、たまに食事を抜いたほうが、俺は丁度いいんです。
-My doctor has warned me not to put on any weight, so it's better for me if I skip a meal every now and then.
+My doctor has actually warned me not to put on any weight,
+ so honestly in my best interests if I skip a meal every now and then.
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_2_1.txt:485]: it's honestly in my best

File: script/Ciel/Day 3/03_01_CIEL03_2_3.txt [hunk 13]
```diff
@@ -342,23 +361,25 @@ She is the very image of a reliable upperclassman scolding an unscrupulous junio
...
 // Lit. expression 'to cut the head off a sleeping person'
-"Come on. If you keep talking like that, you'll get attacked while your guard is down.
+"Come on. If you keep thinking that way, you'll get attacked while your guard is down.
 }
 [sha:f8a7963fc9bad0cd78ed22b3916a451da690998a]{
 -- Page 85, Offset 5052.
 -- ニュースだって三人目の頃から同じコトを言っていましたけど、結局、犯人はまだ捕まっていないじゃないですか」
-The news has been saying the same thing since the third case, but in the end, the culprit hasn't been caught yet, you know."
+The news has been saying the same thing since the third victim was discovered,
+ but the still haven't caught the culprit yet, have they?"
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_2_3.txt:382]: they still

File: script/Ciel/Day 3/03_01_CIEL03_2_3.txt [hunk 14]
```diff
@@ -509,7 +541,7 @@ Noel-sensei is holding a silver accessory in her right hand.
 [sha:dacc7be1717a5c4f995a35e768700bad836e9d81]{
 -- Page 98, Offset 5082.
 -- あれは十字架……だろうか。
-Is that a cross... I wonder.
+It looks like... a cross, maybe.
```
Comment (yk1512) [inline script/Ciel/Day 3/03_01_CIEL03_2_3.txt:544]: just remove the maybe. the like and ... conveys enough uncertainty imo

**Review by yk1512 — APPROVED** [review]

**Pull request merged**